pkgs/development/tools/gdm/default.nix

```diff
@@ -0,0 +1,25 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "gdm-${version}";
+  version = "1.4";
+
+  goPackagePath = "github.com/sparrc/gdm";
+
+  src = fetchgit {
+    url = "https://github.com/sparrc/gdm";
+    sha256 = "0kpqmbg144qcvd8k88j9yx9lrld85ray2viw161xajafk16plvld";
+    rev = version;
```

Comment:
Have you tried `"refs/tags/${version}"`, also is there a reason not to use `fetchFromGitHub` here?